sdk/boms/azure-sdk-bom/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.0.1-beta.1 (Unreleased)
```

Comment:
This happened since the current version listed is this without a changelog entry.

@JimSuplizio should this be skipped since azure-sdk-bom isn't being version managed?

Reply:
@alzimmermsft unfortunately, these changelog entries still need to be correct which is why you're seeing this here.